packages/main/src/TabContainer.ts

```diff
@@ -148,12 +139,9 @@ interface TabContainerTabInOverflow extends CustomListItem {
  * <code>import "@ui5/webcomponents/dist/TabSeparator";</code> (for <code>ui5-tab-separator</code>)
  *
  * @constructor
```

Comment:
getter allItems is treated as readonly property so we have to add default value for it. Since it's array we have to add empty array as default value